Stratis.Bitcoin.IntegrationTests/NodeBuilder.cs

```diff
@@ -76,6 +77,7 @@ public static FullNode BuildFullNode(NodeSettings args)
 				.UseNodeSettings(args)
 				.UseConsensus()
 				.UseBlockStore()
+                .UseIndexStore()
```

Comment:
It would help show up any issues that may be caused by its presence, not that I expect any though. Will remove that.